### PR TITLE
USMCA Certificate of Origin

### PR DESCRIPTION
**Pull request by nissimsan**
COO schema for USMCA trade agreement: https://trade.cbp.gov/USMCA/s/
The schema is directly adapted from FedEx's form: https://www.fedex.com/content/dam/fedex/us-united-states/International/upload/FedEx_USMCA_T-MEC_CUSMA_Fillable_Form.pdf

### CODE REVIEW

File: packages/traceability-schemas/schemas/Person.json [hunk 1]
```diff
@@ -54,6 +54,12 @@
             "title": "Job Title",
             "description": "Person's job title.",
             "type": "string"
+        },
+        "taxId": {
+            "$comment": "{\"term\": \"taxId\", \"@id\": \"https://schema.org/taxID\"}",
+            "title": "Tax Identification Number",
+            "description": "The Tax / Fiscal ID of the organization or person, e.g. the TIN in the US or the CIF/NIF in Spain.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/Person.json:61]:
```suggestion
            "description": "The Tax / Fiscal ID of the organization or person, e.g., the TIN in the US or the CIF/NIF in Spain.",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        },
+        {
+          "type": "array"
+        }
+      ]
+    },
+    "certifier": {
+      "$comment": "{\"term\": \"importer\", \"@id\": \"https://w3id.org/traceability#certifier\"}",
+      "title": "Certifier",
+      "description": "Importer, Exporter or Producer Certification of Origin: Indicate whether the certifier is the exporter, producer or importer in accordance with Chapter 5, Article 5.2 (Claims for Preferential Tariff Treatment) of the agreement.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:25]:
```suggestion
      "description": "Importer, Exporter, or Producer Certification of Origin: Indicate whether the certifier is the exporter, producer, or importer in accordance with Chapter 5, Article 5.2 (Claims for Preferential Tariff Treatment) of the agreement.",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      "enum": ["Importer", "Exporter", "Producer"]
+    },
+    "certifierDetails": {
+      "$comment": "{\"term\": \"certifierDetails\", \"@id\": \"https://w3id.org/traceability#certifierDetails\"}",
+      "title": "Certifier's Details",
+      "description": "Certifier’s name, address, country, e-mail address, telephone number and tax id.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:31]:
```suggestion
      "description": "Certifier’s name, address, country, e-mail address, telephone number, and tax ID.",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      "$ref": "https://w3id.org/traceability/schemas/Entity.json"
+    },
+    "exporterDetails": {
+      "$comment": "{\"term\": \"exporterDetails\", \"@id\": \"https://w3id.org/traceability#exporterDetails\"}",
+      "title": "Exporter's Details",
+      "description": "Exporter’s name, address (including country), email address and telephone number, if different than the certifier. This information is not required if the producer is completing the Certification of Origin and does not know the identity of the exporter. The address of the exporter shall be the place of export of the good in a Party’s territory. ",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:37]:
```suggestion
      "description": "Exporter’s name, address (including country), email address, and telephone number, if different than the certifier. This information is not required if the producer is completing the Certification of Origin and does not know the identity of the exporter. The address of the exporter shall be the place of export of the good in a Party’s territory. ",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      "$ref": "https://w3id.org/traceability/schemas/Entity.json"
+    },
+    "producerDetails": {
+      "$comment": "{\"term\": \"producerDetails\", \"@id\": \"https://schema.org/manufacturer\"}",
+      "title": "Producer's Details",
+      "description": "Producer’s name, address (including country), email address and telephone number, if different than the certifier or exporter, or if there are multiple producers, state “Various” or provide a list of producers. A person that wants this information to remain confidential may state “Available upon request by the importing authorities.” The address of a producer shall be the place of production of the good in a Party’s territory.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:43]:
```suggestion
      "description": "Producer’s name, address (including country), email address, and telephone number, if different than the certifier or exporter, or, if there are multiple producers, state 'Various' or provide a list of producers. A person that wants this information to remain confidential may state 'Available upon request by the importing authorities.' The address of a producer shall be the place of production of the good in a Party's territory.",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+      "type": "array",
+      "items": {
+        "$ref": "https://w3id.org/traceability/schemas/Entity.json"
+      }
+    },
+    "producerConfidential": {
+      "$comment": "{\"term\": \"exporterDetails\", \"@id\": \"https://w3id.org/traceability#producerConfidential\"}",
+      "title": "Producer Confidential",
+      "description": "A person that wants the producer information to remain confidential may state “Available upon request by the importing authorities.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:52]:
```suggestion
      "description": "A person that wants the producer information to remain confidential may state 'Available upon request by the importing authorities.'",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+      "type": "boolean"
+    },
+    "importerDetails": {
+      "$comment": "{\"term\": \"importerDetails\", \"@id\": \"https://w3id.org/traceability#importerDetails\"}",
+      "title": "Importer's Details",
+      "description": "Importer’s name, address, email address and telephone number (if known). The address of the importer shall be in a Party’s territory. If the identity of the importer is unknown, or there are various importer’s, please check the appropriate box.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:58]:
```suggestion
      "description": "Importer’s name, address, email address, and telephone number (if known). The address of the importer shall be in a Party's territory. If the identity of the importer is unknown, or there are various importers, please check the appropriate box.",
```

File: packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+      "type": "array",
+      "items": {
+        "$ref": "https://w3id.org/traceability/schemas/Entity.json"
+      }
+    },
+    "importerUnknown": {
+      "$comment": "{\"term\": \"exporterDetails\", \"@id\": \"https://w3id.org/traceability#importerUnknown\"}",
+      "title": "Importer Unknown",
+      "description": "If the identity of the importer is unknown, or there are various importer’s, please check the appropriate box.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaCertificateOfOrigin.json:67]:
```suggestion
      "description": "If the identity of the importer is unknown, or there are various importers, please check the appropriate box.",
```

File: packages/traceability-schemas/schemas/UsmcaProductSpecifier.json [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
...ood qualifies, as set out in Chapter 4, Article 4.2 (Originating Goods) of the USMCA/T-MEC/CUSMA agreement (https://ustr.gov/sites/default/files/files/agreements/FTA/USMCA/Text/04-Rules-of-Origin.pdf). Criterion A: The good is wholly obtained or produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries, as defined in Article 4.3 of the Agreement; Criterion B: The good is produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries using non-originating materials, provided the good satisfies all applicable requirements of product-specific rules of origin; Criterion C: The good is produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries exclusively from originating materials; or Criterion D: The good is produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries. It is classified with its materials, or satisfies the “unassembled goods” requirement, and meets a regional value content threshold of not less than 60% if the transaction value method is used, or not less than 50% if the net cost method is used (not including RVC for autos); except for goods in Chapter 61-63 of the HTS.",
```
Comment (TallTed) [inline packages/traceability-schemas/schemas/UsmcaProductSpecifier.json:31]:
```suggestion
      "description": "Specify the Origin Criterion under which the good qualifies, as set out in Chapter 4, Article 4.2 (Originating Goods) of the USMCA/T-MEC/CUSMA agreement (https://ustr.gov/sites/default/files/files/agreements/FTA/USMCA/Text/04-Rules-of-Origin.pdf). Criterion A: The good is wholly obtained or produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries, as defined in Article 4.3 of the Agreement; Criterion B: The good is produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries using non-originating materials, provided the good satisfies all applicable requirements of product-specific rules of origin; Criterion C: The good is produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries exclusively from originating materials; or Criterion D: The good is produced entirely in the territory of one or more of the USMCA/T-MEC/CUSMA countries. It is classified with its materials, or satisfies the 'unassembled goods' requirement, and meets a regional value content threshold of not less than 60% if the transaction value method is used, or not less than 50% if the net cost method is used (not including RVC for autos); except for goods in Chapter 61-63 of the HTS.",
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor punctuation, mostly.

**Comment by nissimsan** [comment]
Good stuff @TallTed - thanks! These were all spelling mistakes copied from fedex. :)

**Review by OR13 — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**